graph/patterns/change-status.md

```diff
@@ -0,0 +1,107 @@
+# Change Status & Action Request
+
+Microsoft Graph API Design Pattern
+
+_The change status pattern is an effective alternative to model the ability to trigger side effects while keeping an agnostic, data driven approach_
+
+## Problem
+
+In some cases, exposed resources can trigger side effects that modify the system behavior. For instance, a [`riskyUser`](https://docs.microsoft.com/en-us/graph/api/resources/riskyuser?view=graph-rest-1.0) can be [`confirmedCompromised`](https://docs.microsoft.com/en-us/graph/api/riskyuser-confirmcompromised?view=graph-rest-1.0&tabs=http) or [`dismissed`](https://docs.microsoft.com/en-us/graph/api/riskyuser-confirmcompromised?view=graph-rest-1.0&tabs=http).
+
+These situation are often modeled as a `POST` request to a specific endpoint in the same URI space of the resource, triggering the side effect directly and not returning any data. While it might be a quick and easy solution, there are some considerations:
```

Comment:
POST is not an idempotent operation in Graph thus side effects are expected. here is a ref https://github.com/microsoft/api-guidelines/blob/graph/Guidelines.md#74-supported-methods

Reply:
Idempotency and side effects are two different things. I fail to understand the point you're trying to make

Reply:
https://developer.mozilla.org/en-US/docs/Glossary/Idempotent